src/Exceptions/HttpRequestHeaderFieldsTooLarge.php

```diff
@@ -8,4 +8,3 @@ class HttpRequestHeaderFieldsTooLarge extends HttpException {
     public const DESCRIPTION = "Indicates that the server is unwilling to process the request because its header fields are too large.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/431";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces